### ISSUE TITLE
[frontend] cw aknowledge button text overflow

### ISSUE THREAD

**Issue by fluffye-yay** [gabboman/wafrn#210]
text in button to aknowledge cw overflows vertically, probably only on thin device

![Screenshot_20241229-121812](https://github.com/user-attachments/assets/fcd99d3e-e465-43e7-aa0e-0edc0c7e8a1d)

**Comment by FireIsGood** [comment]
Will probably be changed soon& trade; any suggestions?

**Comment by fluffye-yay** [comment]
you migdt be able te jist reword it to "I understand and want te see this woot" but that might not work on smaller devices either

**Comment by fluffye-yay** [comment]
good now thanks
